cartridges/int_stickyio_sfra/cartridge/controllers/CheckoutServices.js

```diff
@@ -23,18 +23,21 @@ if (stickyioEnabled) {
         server.middleware.https,
         csrfProtection.validateAjaxRequest,
         function (req, res, next) {
+        	var BasketMgr = require('dw/order/BasketMgr');
             var PaymentManager = require('dw/order/PaymentMgr');
             var HookManager = require('dw/system/HookMgr');
             var Resource = require('dw/web/Resource');
             var COHelpers = require('*/cartridge/scripts/checkout/checkoutHelpers');
-
+    		var currentBasket = BasketMgr.getCurrentBasket();
```

Comment:
Please correct spacing issues in line 31?

Reply:
fixed